developer-docs-site/docs/move/prover/spec-lang.md

```diff
@@ -237,7 +237,7 @@ function in such a situation, one can use the notation `::len(v)`.
 In MSL, expressions have partial semantics. This is in contrast to Move program expressions, which
 have total semantics, since they either deliver a value or abort.
 
-An expression `e[X]` that depends on some some variables `X` may have a known interpretation for
+An expression `e[X]` that depends on some variables `X` may have a known interpretation for
```

Comment:
This I believe is generated from source code, so that will need to be updated first, then these would be generated from it.

Reply:
Thanks for an explanation! Anything needed from my side?

Reply:
This might be copied content, but it is in the path that is static and not generated. Good to go.